x-pack/plugins/enterprise_search/public/applications/shared/types.ts

```diff
@@ -30,12 +30,19 @@ export interface IKeaListeners<IKeaActions> {
   actions: IKeaActions;
 }
 
-export interface IKeaReducers<IKeaValues> {
-  [value: string]: [
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    any, // The default state for the value - can be anything
```

Comment:
Ignore last comment. One of the drawbacks of reviewing commit-by-commit

Reply:
For what it's worth the final PR still has an `any`, but I think it's OK to keep it - it makes it easy to do a quick grep for instances in the plugin for where we might (one day, when I understand typescript more 😅) be able to refactor it out.